docs/source/conf.py

```diff
@@ -69,7 +69,7 @@
 
 intersphinx_mapping = {
     "kedro-viz": ("https://docs.kedro.org/projects/kedro-viz/en/v6.6.1/", None),
-    "kedro-datasets": ("https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-2.0.0/", None),
+    "kedro-datasets": ("https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-4.0.0/", None),
```

Comment:
I assume this should be updated? Again, not super familiar with `intersphinx_mapping`; barely looking into it for this investigation.

Because, if update this, sounds like should update a bunch of the other versions, too.

Reply:
While we're at it, maybe we jump to the next release that's about to be published?

Reply:
> Because, if update this, sounds like should update a bunch of the other versions, too.

We can do so in another PR